Palindromic_Array/Palindromic_Array.c

```diff
@@ -0,0 +1,49 @@
+/* A Palindromic Sequence is a sequence of numbers which is 
+the same when read both forward and backward.
+Example: 1, 2, 3, 2, 1 */
+
+#include<stdio.h>
+
+int isPalindromic(int arr[], int N)
```

Comment:
Better to have boolean return type instead so that the function can directly be called inside if statement. Then you can return false if the array is not a palindrome, else true.

Rest looks fine. Well done!